.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -10,6 +10,13 @@ body:
     placeholder: 0.6.1
   validations:
     required: true
+- type: input
+  attributes:
+    label: Rancher Desktop K8s Version
+    description: What version of Kubernetes are you using?
+    placeholder: 1.21.6
```

TO make it look like the other placeholders:

```suggestion
    placeholder: "e.g. 1.21.6"
```